app-cpp/app355.cpp

```diff
@@ -135,6 +136,9 @@ std::vector<OrbitalData> SatOrbit::CalculateOrbitalData(const std::vector<sat355
     }
     else
     {
+        // lock the orbit mutex as both calculate and sort could be multithreaded
+        std::lock_guard<std::mutex> lock(mOrbitMutex);
```

Comment:
what data is this mutex protecting?
I don't see any data members for it to protect.

any mutex locking/unlocking should occur around the data member it's meant to protect